docs/modules/ROOT/pages/runbooks/vshn-redis.adoc

```diff
@@ -0,0 +1,118 @@
+= Redis by VSHN
```

Comment:
It feels like 80% of these runbooks could be put in their own `adoc` file and then included. I think asciidoc also support some variable injection and stuff.

Then we'd have the generic `is everything synced and ready` steps in one place and only have to document the service specific for each service.

Reply:
Refactored the docs